packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -232,6 +232,25 @@ enum ListTileControlAffinity {
 ///
 /// {@tool snippet}
 ///
+/// This example uses an [Expanded] within a [Row] to demonstrate how to
+/// overcome the issue with a [ListTile] attempting to expand horizontally
+/// without proper constraints.
+///
+/// ```dart
+/// Row(
+///   children: const <Widget>[
+///     Expanded(
+///       child: ListTile(
+///         leading: FlutterLogo(),
+///         title: Text('The ListTile is expanded to fill the available space'),
+///       ),
+///     ),
```

Comment:
I would maybe add another widget or two into the `Row` before/after the `ListTile` to make the example more realistic. Having only a single `ListTile` in a `Row` does not really make sense :)